__tests__/extension-requests/extension-requests.test.js

```diff
@@ -197,7 +197,10 @@ describe.skip('Tests the new Extension Requests Screen', () => {
 
     page.on('request', (interceptedRequest) => {
       const url = interceptedRequest.url();
-      if (url === 'https://api.realdevsquad.com/extension-requests') {
+      if (
+        url ===
+        'https://api.realdevsquad.com/extension-requests?size=10&dev=true'
```

Comment:
move it to some constant file

Reply:
This is format followed in all the tests in this repo.